doc/api/next_api_changes/behavior/22347-RQ.rst

```diff
@@ -0,0 +1,13 @@
+Correctly treat pan/zoom events of overlapping axes
+---------------------------------------------------
+
+The forwarding of pan/zoom events is now determined by the visibility of the
+background-patch (e.g. ``ax.patch.get_visible()``) and by the ``zorder`` of the axes.
+
+- Axes with a visible patch do not forward events to axes below.
+  - Only the axis with the highest ``zorder`` that contains the event is triggered
+    (if there are multiple axes with the same ``zorder``, the last added axis counts)
+- Axes with an invisible patch forward events to axes at lower or equal ``zorder``
```

Comment:
> /home/circleci/project/doc/api/next_api_changes/behavior/22347-RQ.rst:9: ERROR: Unexpected indentation.

reStructuredText does not like this formatting. I think l.8-9 do not have to be a bullet point:

```suggestion
- Axes with a visible patch do not forward events to axes below.
  Only the axis with the highest ``zorder`` that contains the event is triggered
  (if there are multiple axes with the same ``zorder``, the last added axis counts)
- Axes with an invisible patch forward events to axes at lower or equal ``zorder``
```

If you really want in indented sublist, you'd have to add blank lines around l.8-9 otherwise ReST doesn't understand it.

Reply:
thanks! lets give it a try 🙂